certz/README.md

```diff
@@ -54,6 +54,41 @@ When no longer a profile is needed it can be removed from the target via
 `Certz.DeleteProfile()` RPC. Note that the gNxI SSL profile cannot be
 removed.
 
+#### Authentication Policy
+
+An authentication policy is a set of rules that defines which CAs can sign which
+certificate. By rotating authentication policies, data center admins can ensure
+that only authorized CAs are able to sign certificates for specific users/roles
```

Comment:
What about "can ensure all endpoints are updated to validate certificates presented by their peers during mutual authentication are signed by one of the authorized CAs in the authentication framework as specified in the policy"?